src/components/OrgSettings/ActionItemCategories/OrgActionItemCategories.tsx

```
@@ -140,18 +151,55 @@
     [openModal],
   );
 
+  /** Apply client-side filtering and sorting */
   useEffect(() => {
-    if (catData && catData.actionItemCategoriesByOrganization) {
-      setCategories(catData.actionItemCategoriesByOrganization);
+    if (catData && catData.actionCategoriesByOrganization) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)